modules/arcgis/src/index.js

```diff
@@ -0,0 +1,5 @@
+const EDL = require("./EsriDeckLayer");
+const EDLV2D = require("./EsriDeckLayerView2D");
+
+export const EsriDeckLayer = EDL.default;

```

Comment:
Q: Why use `require` and not `import` here? That kills tree-shaking, among other things.

Reply:
I had the impression that it would not work, or maybe it really did not work in the beginning (the sample started as a TypeScript app). Anyway, it does work now, and I can drop the `.default`. Thank you!